x/bank/keeper/keeper.go

```diff
@@ -48,6 +48,11 @@ type Keeper interface {
 	DelegateCoins(ctx sdk.Context, delegatorAddr, moduleAccAddr sdk.AccAddress, amt sdk.Coins) error
 	UndelegateCoins(ctx sdk.Context, moduleAccAddr, delegatorAddr sdk.AccAddress, amt sdk.Coins) error
 
+	// StringToBytes decodes text to bytes
+	StringToBytes(text string) ([]byte, error)
+	// BytesToString encodes bytes to text
+	BytesToString(bz []byte) (string, error)
```

Comment:
I wouldn't define these methods on bank keeper. It's totally unclear in this context that they're about addresses. Instead you could make a getter AddressCodec()